README.md

```diff
@@ -105,9 +107,11 @@ dev.off() # reset to default (single) plot window
 #>           1
 ```
 
+### Grouped data
+
 So far, so good. But where `plot2` starts to diverge from its base
 counterpart is with respect to grouped data. In particular, `plot2`
-allows you to characterize groups using the `by` argument.[^1]
+allows you to characterize groups using the `by` argument.\[1\]
```

Comment:
Might be missing some GFM syntax trick, but what does  `\[1\]` give you over `[^1]`?

Reply:
I think my `devtools` is broken